src/gravel/controllers/orch/ceph.py

```diff
@@ -35,6 +37,11 @@
     CephStatusModel
 )
 
+try:
```

Comment:
Please add a comment stating we're expecting only having to miss the module in case of testing on the host. I mean, this is why we're doing this, right?

Reply:
Yes, that is why. Will do!